src/EnergyPlus/FluidProperties.hh

```diff
@@ -666,7 +636,12 @@ struct FluidPropertiesData : BaseGlobalStruct
 
     void clear_state() override
     {
-        new (this) FluidPropertiesData();
+
+        for (int i = 1; i <= refrigs.isize(); ++i) delete refrigs(i);
```

Comment:
Using arrays of pointers to fluid objects, so remember to delete in `clear_state`.